Makefile

```diff
@@ -12,11 +12,12 @@ JS_FILES += ${IHP}/static/vendor/turbolinksMorphdom.js
 
 include ${IHP}/Makefile.dist
 
-tailwind-dev:
+tailwind-dev: node_modules
```

Comment:
@mpscholten what is `node_modules` doing here?

Reply:
Oh, I guess it should be `npm`

so it calls `make npm`

Reply:
It triggers a `make node_modules` when the `node_modules` directory doesn't exists

Reply:
so we should define also: 

```
node_modules:
     npm install
```

?

Reply:
It's already defined like that (npm ci is similar to bpm install)

Reply:
Thanks, moving discussion to https://github.com/digitallyinduced/ihp/pull/1890